docs/run_rackhd/docker_based_installation.rst

```diff
@@ -3,8 +3,63 @@ Installation from Docker
 
 .. contents:: Table of Contents
 
+Prerequisites
+-----------------------------
+
+**NICs**
+
+.. tabs::
+
+    .. tab:: Ubuntu 14.04
+
+        Start with an Ubuntu trusty(14.04) instance with 2 nics:
+
+        * ``eth0`` for the ``public`` network - providing access to RackHD APIs, and providing
+          routed (layer3) access to out of band network for machines under management
+
+        * ``eth1`` for dhcp/pxe to boot/configure the machines
+
+        edit the network:
+
+        * ``eth0`` - assign IP address as appropriate for the environment, or you can use DHCP
+
+        * ``eth1`` static ( 172.31.128.0/22 )
+
+        please check the network config file: ``/etc/network/interfaces``. The ``eth1``'s ip address is ``172.31.128.1`` Like as follows:
+
+        .. code::
+
+            auto eth1
+            iface eth1 inet static
+            address 172.31.128.1
+            post-up ifconfig eth1 promisc
+
+    .. tab:: Ubuntu 16.04
+
+        Start with an Ubuntu xenial(16.04) instance with 2 nics:
+
+        * ``ens160`` for the ``public`` network - providing access to RackHD APIs, and providing
+          routed (layer3) access to out of band network for machines under management
+
+        * ``ens192`` for dhcp/pxe to boot/configure the machines
```

Comment:
ens160 and ens192 might not be fixed as it's related to NIC's physical positions in the pci system after Ubuntu 15.04.

Reply:
agreed, let me add notes for the nic name.

Reply:
@iceiilin , added additional notes for ethernet name, please review again, thanks~~

Reply:
Same problem with eth0 and eth1. Here eth0/eth1 or ens160/ens192 are just examples.